src/microbe_stage/MicrobeCheatMenu.cs

```diff
@@ -26,29 +29,43 @@ public class MicrobeCheatMenu : CheatMenu
     [Export]
     public NodePath DespawnAllEntitiesPath = null!;
 
+    [Export]
+    public NodePath SetTimePath = null!;
+
+    [Export]
+    public NodePath TargetTimePath = null!;
+
 #pragma warning disable CA2213
     private CustomCheckBox infiniteCompounds = null!;
     private CustomCheckBox godMode = null!;
     private CustomCheckBox disableAI = null!;
+    private CustomCheckBox lockTime = null!;
     private Slider speed = null!;
     private Button playerDivide = null!;
     private Button spawnEnemy = null!;
     private Button despawnAllEntities = null!;
+    private Button setTime = null!;
+    private SpinBox targetTime = null!;
```

Comment:
Why not have a slider that when changed would change the game time to that? That would allow very easy scrubbing of the time of day to easily visualize time changing shaders etc.